pkg/virt-config/featuregate/active.go

```diff
@@ -104,4 +108,5 @@ func init() {
 	RegisterFeatureGate(FeatureGate{Name: VirtIOFSConfigVolumesGate, State: Alpha})
 	RegisterFeatureGate(FeatureGate{Name: VirtIOFSStorageVolumeGate, State: Alpha})
 	RegisterFeatureGate(FeatureGate{Name: DecentralizedLiveMigration, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: DeclarativeHotplugVolumesGate, State: Alpha})
```

Comment:
I wonder how upgrade will look like for users when this goes beta/GA **and** they've had the legacy hotplug FG.

I guess nothing happens and they just don't get the new feature unless they did some research and disabled the legacy one?

Regarding kubevirt CSI (or other projects piggybacking on kubevirt), it would be a good idea to proactively switch? Probably a little early now but this would justify a release note highlighting that

Reply:
It was originally doing that, but we have switched to persistent hotplugging recently, so it is using the VM endpoint, not the VMI one.

Reply:
> It was originally doing that, but we have switched to persistent hotplugging recently, so it is using the VM endpoint, not the VMI one.

Glad to hear! I was vaguely remembering using ephemeral hotplugs

I would say there's no rush to cutover and the subresource API is not going anywhere regardless